PrivateHeaders/APPX/ZIP.h

```diff
@@ -325,6 +372,9 @@ namespace appx {
            << "xmlns=\"http://schemas.microsoft.com/appx/2010/blockmap\" "
            << "HashMethod=\"http://www.w3.org/2001/04/xmlenc#sha256\">";
         for (const ZIPFileEntry &entry : otherEntries) {
+            if (_IsAPPXFile(entry.fileName)) {
```

Comment:
This should be conditional on whether we are building a bundle or not.

IMO the caller shouldn't pass .appx files via `otherEntries` if they shouldn't end up in the block map.

Add a comment?